.github/workflows/ci.yml

```diff
@@ -53,9 +56,20 @@ jobs:
       - name: Compile and test flink-connector-elasticsearch
         timeout-minutes: 20
         run: |          
-          mvn clean install -U -B --no-transfer-progress \
+          set -o pipefail
+
+          mvn clean deploy ${MVN_COMMON_OPTIONS} \
+            -DaltDeploymentRepository=validation_repository::default::file:${{ env.MVN_VALIDATION_DIR }} \
             -Dscala-2.12 \
             -Prun-end-to-end-tests -DdistDir=$(pwd)/../flink-1.16-SNAPSHOT \
             -Dflink.convergence.phase=install -Pcheck-convergence \
             ${{ env.MVN_CONNECTION_OPTIONS }} \
+            -Dlog4j.configurationFile=file://$(pwd)/tools/ci/log4j.properties \
+            | tee ${{ env.MVN_BUILD_OUTPUT_FILE }}
+
+      - name: Check licensing
```

Comment:
Thought if we should consider moving the license checker earlier but I think there's little value in that, given the amount of time it takes for a testrun to complete.

Reply:
We could only do that if we again split of the compile & test steps (costing us time), and I'd like to avoid that as long as we are running on ASF infrastructure.